.github/workflows/cut-release-branch.yml

```diff
@@ -293,8 +293,7 @@ jobs:
         run: |
           source env/bin/activate
           python -m pip install -r dev-requirements.txt
-          env/bin/bumpversion --allow-dirty \ 
-           --new-version ${{ needs.create_temp_branch.outputs.next-version }} major
```

Comment:
I'm thinking this prior syntax could have worked with some indentation but want to unblock the ongoing release with a simple fix for the time being.